specs/mediawiki/v1/content.yaml

```diff
@@ -1142,10 +1143,17 @@ paths:
           type: integer
           required: true
         - name: sections
-          in: formData
+          in: body
           description: Sections to transform
           type: string
           required: true
+          schema:
+            type: 'object'
+            properties:
+              sections:
```

Comment:
@d00rman right now we can't put a `string` there. Because we still support both `application/json` and `form-data` on this endpoint. In first case this will be an object, in the second case - a string. So, we first need to properly deprecate the `form-data` option, ensure nobody's using it, and only then we can set `type: object` here. 

As for description - yes, we could, but it wouldn't be seen anywhere in the swagger UI.